moskito-backend/src/Controller/CampsiteController.php

```diff
@@ -66,4 +60,57 @@ public function create(
                 true
             );
     }
+
+    /**
+     * @Route("/campsite-filter", methods={"POST"})
+     */
+
+    public function filter(
+        Request $request,
+        CampsiteRepository $campsiteRepository,
+        CampsiteFeatureRepository $featureRepository,
+        CampsiteFeatureSerializer $featureSerializer,
+        CampsiteSerializer $campsiteSerializer
+        ): JsonResponse {
+            $filteredFeatures = $featureSerializer->deserialize($request->getContent());
+            $filteredCampsites = [];
+
+            foreach($filteredFeatures as $filteredFeature) {
```

Comment:
oha das sind viele schleifen. Solche Business Logik bitte nicht im Controller, sondern in eigene "Services". 
Bitte teile die Schleifen auf - versuche (private Teil-)Funktionen zu bauen.